README.md

```diff
@@ -98,7 +110,7 @@ Error from server (Forbidden): admission webhook "validation.gatekeeper.sh" deni
 
 You just validated the container images in your k8s cluster!
 
-- Uninstall Ratify
+Step 4: Uninstall Ratify
```

Comment:
for `helm delete ratify`, we might need to add the `gatekeeper-system` namespace as well since it's default now.

Reply:
fixed. thanks!